pom.xml

```diff
@@ -105,7 +105,7 @@
             <configuration>
               <rules>
                 <requireJavaVersion>
-                  <version>[1.7.0-0,1.9.0-0)</version>
+                  <version>[1.7.0-0,1.9.0-0),[9.0.0,11.0.5]</version>
```

Comment:
When running `mvn install -Dgpg.skip` with `openjdk 12.0.2` I get:
```
INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (validate) @ java-dogstatsd-client ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireJavaVersion failed with message:
Detected JDK Version: 12.0.2 is not in the allowed range [1.7.0-0,1.9.0-0),[9.0.0,11.0.5].
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.956 s
[INFO] Finished at: 2020-01-13T17:12:17+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (validate) on project java-dogstatsd-client: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed.
```

If I replace `[9.0.0,11.0.5]` by `[9.0.0,12.0.5]`, `mvn install -Dgpg.skip` succeed.

Note: `mvn install -Dgpg.skip` succeed on master branch without any changes.

Reply:
Yeah, that's because Java12 was not supported java version. I can bump to Java12, but it shouldn't be a problem as we'll be building on Java8 most likely targeting Java7, so I'm not too concerned, but I'll bump this. If it works on Java12 no reason not to support it.